packages/discovery-provider/src/tasks/index_core_cutovers.py

```diff
@@ -0,0 +1,33 @@
+from src.utils.config import shared_config
+
+env = shared_config["discprov"]["env"]
+
+# dev plays
+DEV_PLAYS_SOL_CUTOVER = 0
+DEV_PLAYS_CORE_CUTOVER = 0
+
+# stage plays
+STAGE_PLAYS_SOL_CUTOVER = 308491544
+STAGE_PLAYS_CORE_CUTOVER = 10000
```

Comment:
maybe we want to rollover stage and start from 0 here?

Reply:
i would say we should do on stage what we plan to do on prod